staging/src/k8s.io/apiserver/pkg/cel/mutation/typeprovider.go

```diff
@@ -60,6 +60,12 @@ func (p *TypeProvider) FindStructType(structType string) (*types.Type, bool) {
 	return p.underlyingTypeProvider.FindStructType(structType)
 }
 
+// FindStructFieldNames returns the field names associated with the type, if the type
+// is found.
+func (p *TypeProvider) FindStructFieldNames(structType string) ([]string, bool) {
+	return nil, true
```

Comment:
Does this stub implementation saying every type is found with no fields affect any behavior?

Reply:
Nope. We do not enumerate members so this is unused.